settings.gradle.kts

```diff
@@ -34,56 +34,61 @@ plugins {
 
 include(":hedera-node")
 
-include(":hedera-node:hedera-networkadmin-service")
+include(":node-app-service-networkadmin", "hedera-networkadmin-service")
```

Comment:
The approach only works if the project names align with the Module Names. This is good for consistency anyway in my opinion. Ideally, we would also rename the project folders. Right now, here we define the (now changed) project name and then the (unchanged) folder.